test/image.bats

```diff
@@ -136,6 +136,7 @@ function teardown() {
 }
 
 @test "image pull with signature" {
+	skip "registry has some issues"
```

Comment:
It's too bad there's not a "Skip with warning" type of skip.  It would be nice to run this in general.